src/wmnds/components/content-tiles/small2/_small2.scss

```
@@ -0,0 +1,7 @@
+.wmnds-content-tile__copy {
```

For good practice, change to...
```suggestion
.wmnds-content-tile {
    &__copy {
```